Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN apt-get update -y && \
 	apt-get install -y \
 		git \
 		unzip && \
-	curl -Ls -o /usr/bin/k8s-claimer https://storage.googleapis.com/k8s-claimer/git-e4dcc16/k8s-claimer-git-e4dcc16-linux-amd64 && \
```

Comment:
What's the reasoning behind moving this from an immutable version to a mutable one? I'm not sure I'm a fan of the potential increased surface for future failz.